### DISCUSSION TITLE
ThisIsWin11! Yes or No for changing the appname?

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/ThisIsWin11#40] [Announcements]
Share your thoughts here!

Vote here!
https://www.builtbybel.com/blog/19-apps/44-thisiswin11-yes-or-no-for-changing-the-appname

**Comment by f00tf00t** [comment]
WXI-T
or
XI Tool

short and to the point

**Comment by alenmeister** [comment]
`Nerf11` from the comments on the blog post is a solid suggestion!
